crates/kubelet/src/container/mod.rs

```diff
@@ -138,22 +138,22 @@ impl Container {
     }
 
     /// Get arguments of container.
-    pub fn args(&self) -> &Vec<String> {
+    pub fn args(&self) -> &Option<Vec<String>> {
```

Comment:
In the original API all of these functions were `Option<&T>` rather than borrowed options. Mind switching the signatures in this file to use that instead with `as_ref`?

Reply:
I don't understand. AFAIK, it was the original signature before 4846df7:

https://github.com/krustlet/krustlet/blob/da85061ec2ef809590cad023ecdefc6891102c34/crates/kubelet/src/container/mod.rs#L141-L143

Reply:
I persist :smiley:: you **did** have `args` in 0.7, and its return type was `&Option<Vec<String>>`: cf https://github.com/krustlet/krustlet/blob/v0.7.0/crates/kubelet/src/container/mod.rs#L141-L143

However, no problem to change it and have the borrowed inner type. I'm updating the PR.

Reply:
Oh that was in container 🤦 . Sorry. We should have caught that earlier. Thanks for doing it anyway!